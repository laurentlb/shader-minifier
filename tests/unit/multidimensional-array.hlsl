// All different sizes will not be merged
float a[4][5];
float b[SIZE_X][SIZE_Y];
UserT c[SIZE_X][3];

// Neighboring declarations with the same size will be merged
float d[4][3];
float e[4][3];

// Non-neighboring mergable declarations are not merged
float f[5][2];
float g[9][15];
float h[5][2];