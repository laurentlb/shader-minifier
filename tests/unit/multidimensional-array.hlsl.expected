float a[4][5];
float b[SIZE_X][SIZE_Y];
UserT c[SIZE_X][3];
float d[4][3],e[4][3];
float f[5][2];
float g[9][15];
float h[5][2];
